src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java

```diff
@@ -111,8 +112,18 @@
    * @return [--name, file.getAbsolutePath()] or [] if file=null.
    */
   static List<String> filePath(String name, File file) {
-    if (file != null && !Strings.isNullOrEmpty(file.getAbsolutePath())) {
-      return Arrays.asList("--" + name, file.getAbsolutePath());
+    if (file != null) {
+      return path(name, file.toPath());
+    }
+    return Collections.emptyList();
+  }
+
+  /**
+   * @return [--name, path.toString()] or [] if path is null or not set.
```

Comment:
I know that this code is probably going away and the rest of the code in this class doesn't follow this convention, but just an FYI that this isn't actually Google style.  Checkout the [style guide](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)

You should also use {@code } for any values, code, arrays, etc.